expression/builtin_time.go

```diff
@@ -283,6 +288,58 @@ func (b *builtinDateSig) evalTime(row []types.Datum) (types.Time, bool, error) {
 	return expr, false, nil
 }
 
+type dateLiteralFunctionClass struct {
+	baseFunctionClass
+}
+
+func (c *dateLiteralFunctionClass) getFunction(ctx context.Context, args []Expression) (builtinFunc, error) {
+	if err := c.verifyArgs(args); err != nil {
+		return nil, errors.Trace(err)
+	}
+	constant, ok := args[0].(*Constant)
+	if !ok {
+		return nil, errors.Trace(types.ErrInvalidTimeFormat)
```

Comment:
should we handle this error to respect to sql mode ?

Reply:
nope.